### DISCUSSION TITLE
Sharing prisma database between multiple servers

### DISCUSSION THREAD

**Discussion by kingfishx23** [prisma/prisma#10100] [Q&A]
Hi, I'm wondering how to do this since prisma is generated per project

should I duplicate the prisma folder? or is there a way to code share?

**Comment by austincrim** [comment]
Hey @kingfishx23 👋,

Could you help me understand what you mean exactly? Are you wanting to use a single Prisma schema file across multiple projects? Or would you like query from multiple data sources from one schema?

**Reply by kingfishx23** [reply]
lets say i have multiple servers (projects)
1. general all-purpose server
2. specific data server that serve 

and i have 1 postgres database

i want the server to be able to use prisma as the query engine. how should i do this?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
